js/mainScript.js

```diff
@@ -27,17 +25,13 @@ function removeBook(e, newBookElement) {
   bookArray = bookArray.filter(checkBtnclicked);
   newBookElement.remove();
 }
-let entireJSON = '';
+
+const entireJSON = localStorage.getItem('bookKey');
 function syncStorage() {
-  entireJSON = localStorage.getItem('bookKey');
   if (entireJSON != null) {
-    localStorage.setItem(
-      'bookKey',
-      entireJSON.concat(JSON.stringify(bookArray)),
-    );
+    localStorage.setItem('bookKey', entireJSON.concat(JSON.stringify(bookArray)));
   } else {
```

Comment:
- [ ] The data is preserved in the local storage after adding books. Great work 👏. However, the data should also be loaded from the local storage and displayed on the page after opening the page or refreshing the page. Kindly work on that so that your project is complete.

Reply:
Okay, thank you we will make the corrections ASAP, I think the project requirement did not state this clearly because me and my teammates were actually on different understanding. According to the project requirement this was stated " Make sure that data is preserved in the browser's memory by using localStorage." unlike what various reviewers mentioned "displayed on the page after opening the page or refreshing the page."